### DISCUSSION TITLE
The afterCreate-hook in a relationManager does not have access to the created record

### DISCUSSION THREAD

**Discussion by Kobo-one** [filamentphp/filament#2528] [Help] [answered]
### Package

filament/filament

### Package Version

v2.12.14

### Laravel Version

v9.13.0

### Livewire Version

v2.10.5

### PHP Version

PHP 8.1.0

### Bug description

When using the afterCreate hook in a relationManager you cannot access the newly created record. 
When creating a record using a regular create page you have access to `$this->record`.

It looks like there is a difference between the create method in `Filament\Resources\Pages\CreateRecord` (page) and the create method in `Filament\Resources\RelationManagers\Concerns\CanCreateRecords` (relationManager).
In the latter the result of `$this->handleRecordCreation($data)` (the newly created record) is stored in a local variable instead of a property. 

My proposal is to use the same logic as `CreateRecord::create` and store the record in a property instead.

I would be happy to provided a pull-request if this looks like a relevant change.

**Comment by danharrin** [comment, marked as answer]
`$this->getMountedTableActionRecord()`
